### PR TITLE
mirrors/index: Add "Creating a mirror" subsection.

### PR DESCRIPTION
**Pull request by flexibeast**
Addresses #392.

### CODE REVIEW

File: src/xbps/repositories/mirrors/index.md [hunk 1]
```diff
@@ -53,3 +53,12 @@ sub-repository.
...
 Mirrors](./tor.md) for more information.
+
+## Creating a mirror
+
+If you'd like to set up a mirror, follow one of the many guides available (e.g.
+[this EFF
+guide](https://www.eff.org/keeping-your-site-alive/mirroring-your-site)), then
+submit a pull request to [the void-docs
+repository](https://github.com/void-linux/void-docs) to add your mirror to this
+page. A mirror requires at least 1.1TB of storage space.
```
Comment (ericonr) [inline src/xbps/repositories/mirrors/index.md:64]: "a full mirror requires around 1TB of storage", to allow for increases. Does adding "full" there help with making it clear that the mirror can have less than all the repositories?

Comment (flexibeast) [reply src/xbps/repositories/mirrors/index.md]: Oh, well, i thought that "at least" covered the possibility of increases. :-) But i'll change it. And yes, adding "full" is good, i think. PR updated.

**Comment by ericonr** [comment]
There's also this answer from @the-maldridge in one of the issues about mirrors:

> Running a mirror is pretty straightforward. All you have to do is setup rsync and keep it reasonably up to date. If your mirror is going to be exposed outside of your region, this should be noted in the documentation. If your mirror is internal to your region, i.e. behind some kind of carrier NAT, then this should also be noted.
> Please keep in mind that we pay bandwidth for all data sent out from the T1 mirrors. You can respect this by only mirroring if your use case for your mirror will offset the network throughput consumed by your mirror syncing.

I think we should try to include some of it.

**Comment by flexibeast** [comment]
Thanks - i've incorporated that information and updated the PR. One thing to note is that the tables on this page currently don't include information about whether or not a mirror is only internal to a region.

File: src/xbps/repositories/mirrors/index.md [hunk 2]
```diff
@@ -53,3 +53,21 @@ sub-repository.
...
+
+If you'd like to set up a mirror, and are confident you can keep it reasonably
+up-to-date, follow one of the many guides available for mirroring with
+[rsync(1)](https://man.voidlinux.org/rsync.1), then submit a pull request to
+[the void-docs repository](https://github.com/void-linux/void-docs) to add your
+mirror to this page.
+
+If your mirror is going to be exposed outside of your region, this should be
+noted in the documentation. If your mirror is internal to your region, i.e.
+behind some kind of carrier NAT, this should also be noted.
```
Comment (ericonr) [inline src/xbps/repositories/mirrors/index.md:67]: @the-maldridge would you be fine with changing this to apply only for mirrors that are internal to their region? Most of the mirrors seem to be available worldwide (or at least from Brazil), so not being available everywhere would be the exception.

File: src/xbps/repositories/mirrors/index.md [hunk 3]
```diff
@@ -53,3 +53,21 @@ sub-repository.
...
+
+A full mirror requires around 1TB of storage.
+
+Please keep in mind that we pay bandwidth for all data sent out from the Tier 1
+mirrors. You can respect this by only mirroring if your use case for your mirror
+will offset the network throughput consumed by your mirror syncing.
```
Comment (ericonr) [inline src/xbps/repositories/mirrors/index.md:73]: "It is also possible to mirror only part of the repositories. Excluding debug packages is one way of decreasing the load on the Tier 1 mirros that has low impact on users."

**Comment by ericonr** [comment]
> One thing to note is that the tables on this page currently don't include information about whether or not a mirror is only internal to a region.

IIRC, the only mirror I couldn't access was `hushan.tech`.

File: src/xbps/repositories/mirrors/index.md (hunk 3 above)
Comment (flexibeast) [reply src/xbps/repositories/mirrors/index.md:75]: Added to the previous para, where it seems to fit best, with some small changes.

**Comment by ericonr** [comment]
>     maldridge │ then there should be a new section of region-locked mirrors

So a separate table for region locked Tier 2 mirrors.

**Comment by flexibeast** [comment]
@ericonr: So create a new table with the hushan.tech mirror as the only entry? Or should we first confirm that it is indeed region-locked, rather than not being available for some other reason? If the latter, i'd prefer to merge this PR first, and open a separate issue about the hushan.tech mirror.

**Comment by the-maldridge** [comment]
@flexibeast new table; it is region locked.

**Comment by flexibeast** [comment]
@the-maldridge: Thanks, PR updated.

Is there anything else, or is this now ok to merge?

File: src/xbps/repositories/mirrors/index.md (hunk 2 above)
Comment (ericonr) [reply src/xbps/repositories/mirrors/index.md:74]: I think we need to update this paragraph then, right?

Comment (flexibeast) [reply src/xbps/repositories/mirrors/index.md]: Gah, yes, thanks. i've removed this para entirely, and added the relevant text to the preceding one. WDYT?

Comment (ericonr) [reply src/xbps/repositories/mirrors/index.md]: Looks good to me, thanks!

**Pull request merged**